src/Psy/Configuration.php

```diff
@@ -115,6 +118,10 @@ public function init()
         if ($configFile = $this->getConfigFile()) {
             $this->loadConfigFile($configFile);
         }
+
+        if (!$this->configFile && $localConfig = $this->getLocalConfigFile()) {
```

Comment:
@bobthecow now skips loading local config if the user passed a config file option.